.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,92 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Weekly Wednesday 7 AM build
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 7 * * 3'
+
+env:
+  TEST_BIGDATA: "https://bytesalad.stsci.edu/artifactory"
+
+jobs:
+  pep:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Lint with flake8
+      run: |
+        python -m pip install --upgrade pip flake8
+        flake8 ci_watson --count
+    # Make sure that packaging will work
+    - name: pep517 build
+      run: |
+        python -m pip install --upgrade setuptools build "twine>=3.3"
+        python -m build --sdist .
+        twine check --strict dist/*
+
+  basic_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+    - name: Install and build
```

Comment:
Any reason we're running under `python 3.7`?  Maybe we should run under a matrix of python versions?

Reply:
I thought Astroconda only works in Python 3.7 and JenkinsfileRT uses Astroconda.

Reply:
Between the three jobs, Python 3.7-3.9 are tested here. Making it fancy with matrix/tox can be future work.